flickit-assessment-data/src/main/resources/liquibase/changelog-0.36.xml

```diff
@@ -65,4 +65,21 @@
                              constraintName="uq_fac_attribute_value_assessmentresultid_attributeid"
                              deferrable="true" initiallyDeferred="true"/>
     </changeSet>
+
+    <changeSet id="0.36-07" author="Maziyar Gerami">
+        <addColumn tableName="fak_kit_version">
+            <column name="status_version" type="bigint" valueComputed="
+            CASE
+                WHEN status = 0 THEN 0
+                WHEN status = 1 THEN 1
+                ELSE -id
+            END"/>
```

Comment:
**Suggestion:** Ensure that the `status_version` column addition and unique constraint in the Liquibase changelog handle all potential edge cases and statuses correctly. [general, importance: 5]
```suggestion
            <changeSet id="0.36-07" author="Maziyar Gerami">
                <addColumn tableName="fak_kit_version">
                    <column name="status_version" type="bigint" valueComputed="
                    CASE
                        WHEN status = 0 THEN 0
                        WHEN status = 1 THEN 1
                        WHEN status = 2 THEN 2  // Assuming status 2 needs handling
                        ELSE -id
                    END"/>
                </addColumn>
            </changeSet>
```